node/pkg/driver/device_connectivity/device_connectivity_helper_scsigeneric.go

```diff
@@ -44,6 +45,10 @@ type OsDeviceConnectivityHelperScsiGenericInterface interface {
 	FlushMultipathDevice(mpathDevice string) error
 	RemovePhysicalDevice(sysDevices []string) error
 	ValidateLun(lun int, sysDevices []string) error
+	GetVolumeIdVariations(volumeUuid string) []string
+	GetMatchingVolumeIdToMpathName(mpathdOutput string, mpathDeviceName string, volumeIdVariations []string) (string, error)
+	GetMpathdOutputByVolumeIds(volumeIdVariations []string) (string, error)
+	GetMpathDeviceName(volumePath string) (string, error)
```

Comment:
```suggestion
	GetVolumeIdVariations(volumeUuid string) []string
	GetMatchingVolumeIdToMpathName(mpathdOutput string, mpathDeviceName string, volumeIdVariations []string) (string, error)
	GetMpathdOutputByVolumeIds(volumeIdVariations []string) (string, error)
	GetMpathDeviceName(volumePath string) (string, error)
```
```suggestion
	GetVolumeIdVariations(volumeUuid string) []string
	GetVolumeIdOfMpathName(mpathdOutput string, mpathDeviceName string, volumeIdVariations []string) (string, error)
	GetMpathdOutputForVolume(volumeIdVariations []string) (string, error)
	GetMpathDeviceName(volumePath string) (string, error)
```
btw why are these added to this interface and not directly into the helper (non-scsi) generic?

Reply:
because we already [call](https://github.com/IBM/ibm-block-csi-driver/blob/404803e569d4ac8cf3f0912719eec9ae49883d0f/node/pkg/driver/node_utils.go#L101) the `scsi` helper, so I didn't want to add another parameter in the `nodeutils` struct, do you think I should change it?

Reply:
done